pkg/server/container_execsync.go

```diff
@@ -127,9 +119,7 @@ func (c *criContainerdService) execInContainer(ctx context.Context, id string, o
 		return nil, fmt.Errorf("failed to create exec %q: %v", execID, err)
 	}
 	defer func() {
-		// TODO(random-liu): There is a containerd bug here containerd#1376, revisit this
-		// after that is fixed.
-		if _, err := process.Delete(ctx); err != nil {
+		if _, err := process.Delete(ctx, containerd.WithProcessKill); err != nil {
```

Comment:
Should not `WithProcessKill` here, it will fail and skip delete if `Exec` is not started yet.